app/scripts/modules/core/src/managed/config/Configuration.tsx

```diff
@@ -0,0 +1,127 @@
+import cx from 'classnames';
```

Comment:
nit: For consistency, we usually call import this as: `import classNames from 'classnames'`